bentoml/_internal/service/inference_api.py

```diff
@@ -45,9 +45,11 @@ def __init__(
         self.doc = doc
         self.route = route
 
-    def __repr__(self):
-        return "<{}({} -> {})>".format(
-            self.__class__.__name__, repr(self.input), repr(self.output)
+    def __str__(self):
+        return u"%s(%s\u2192 %s)" % (
+            self.__class__.__name__,
+            str(self.input),
+            str(self.output),
         )
```

Comment:
I feel like f strings are more readable than formats like this.

Also, why not just have the actual unicode character?

Reply:
sure I will change it to f-string